src/cloudflare/internal/test/vectorize/vectorize-mock.js

```diff
@@ -114,7 +114,7 @@ export default {
           returnSet.forEach((v) => {
             delete v.values;
           });
-        if (!body?.returnMetadata)
+        if (!body?.returnMetadata || body?.returnMetadata === "none")
```

Comment:
```suggestion
         if ((body?.returnMetadata || 'none') === "none")
```

nit: Might be a bit cleaner